pkg/resources/stream_acceptance_test.go

```diff
@@ -40,6 +40,18 @@ func TestAcc_Stream(t *testing.T) {
 					checkBool("snowflake_stream.test_stream", "show_initial_rows", false),
 				),
 			},
+			{
+				Config: externalTableStreamConfig(accName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_stream.test_stream", "name", accName),
+					resource.TestCheckResourceAttr("snowflake_stream.test_stream", "database", accName),
+					resource.TestCheckResourceAttr("snowflake_stream.test_stream", "schema", accName),
+					resource.TestCheckResourceAttr("snowflake_stream.test_stream", "on_table", fmt.Sprintf("%s.%s.%s", accName, accName, "STREAM_ON_TABLE")),
```

Comment:
Is this supposed to be `on_external_table`?

Reply:
And I think the name is supposed be `accName`, as well... not `STREAM_ON_TABLE`

Reply:
Hey @alldoami `on_external_table` is a `bool`, and creating a stream on an external table without `... EXTERNAL TABLE` will raise an error; so, the acceptance test is simply ensuring that the table referenced by `on_table` (which is an external table created via `externalTableStreamConfig`) exists. 

That said, I should definitely have added a check that `on_external_table` is `true` - see the most recent commits, which also resolve the `STREAM_ON_TABLE` reference - thanks! I'll have to set up an environment to run these tests locally which will reduce impact on you via these PRs, sorry.